docs/explanations/status_columns.md

```diff
@@ -101,3 +101,98 @@ allow_beta: True
 listed, even if it is invalid or typed the wrong way. Those appear in `status`
 output for informational and debugging purposes.
+
+## Machine Readable Output
+
+The `pro status` command supports a `--format` flag with options including `json` and `yaml`. These result in a machine-readable form of the information presented in the `pro status` command.
+
+```{note}
+`pro status` should return the same results with and without sudo/root, but earlier versions did not always and we would still recommend using sudo if possible.
+```
+
+For example, running `sudo pro status --format=json` on an attached machine may give you something like this:
+```javascript
+{
+  "_doc": "Content provided in json response is currently considered Experimental and may change",
+  "_schema_version": "0.1",
+  "account": {
+    "created_at": "2000-01-02T03:04:05+06:00",
+    "id": "account_id",
+    "name": "Test"
+  },
+  "attached": true,
+  "config": { ...effectiveConfiguration },
+  "config_path": "/etc/ubuntu-advantage/uaclient.conf",
+  "contract": {
+    "created_at": "2000-01-02T03:04:05+06:00",
+    "id": "contract_id",
+    "name": "contract_name",
+    "products": [ "uaa-essential" ],
+    "tech_support_level": "essential"
+  },
+  "effective": null,
+  "environment_vars": [...proClientEnvironmentVariables],
+  "errors": [],
+  "execution_details": "No Ubuntu Pro operations are running",
+  "execution_status": "inactive",
+  "expires": "9999-12-31T00:00:00+00:00",
+  "features": {},
+  "machine_id": "machine_id",
+  "notices": [],
+  "result": "success",
+  "services": [
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Expanded Security Maintenance for Applications",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "esm-apps",
+      "status": "enabled",
+      "status_details": "Ubuntu Pro: ESM Apps is active"
+    },
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Expanded Security Maintenance for Infrastructure",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "esm-infra",
+      "status": "enabled",
+      "status_details": "Ubuntu Pro: ESM Infra is active"
+    },
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Canonical Livepatch service",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "livepatch",
+      "status": "enabled",
+      "status_details": ""
+    },
+    ...otherServiceStatusObjects
+  ],
+  "simulated": false,
+  "version": "27.13.6~18.04.1",
+  "warnings": []
+}
+```
+
+Some particularly important attributes in the output include:
+* `attached`: This boolean value indicates whether this machine is attached to an Ubuntu Pro account or not. Note that this does not imply that any particular service (e.g. esm-infra) is enabled. You must check the individual service object for that status.
```

Comment:
```suggestion
* `attached`: This boolean value indicates whether or not this machine is attached to an Ubuntu Pro account. This does not tell you if any particular service (e.g. `esm-infra`) is enabled. You must check the individual service object for that status.
```

Reply:
As a quick note, I'm not sure about the wording for the final sentence - I've not heard of a "service object" before so I'm not sure what the sentence means. Is there a clearer way we can express this?

Reply:
I reworded it - let me know if it makes more sense now.